packages/graphback-codegen-schema/src/transformer/SchemaTransformer.ts

```diff
@@ -0,0 +1,10 @@
+import { GraphQLSchema } from "graphql";
+
+interface SchemaTransformer<TransformerConfig = any> {
+
+    /**
+     * Performs transformation on the schema and returns target schemaƒ
+     * @param schema 
+     */
+    transformSchema?(schema: GraphQLSchema, config: TransformerConfig): GraphQLSchema;
```

Comment:
Maybe some sort of high level diagram to new generator that shows where this interface fits in?

Reply:
I will need to commit other files in core. Target of this PR is to focus on schema only, but I'm going to create separate PR for the engine as well.

Reply:
Working on diagra now and attaching it to the issue

Reply:
Done. Applied to the issues instead of PR